WcaOnRails/config/routes.rb

```diff
@@ -91,9 +91,12 @@
   get 'admin/delegates' => 'delegates#stats', as: :delegates_stats
 
   get 'robots' => 'static_pages#robots'
+
   get 'server-status' => 'server_status#index'
-  get 'edit-translation' => 'git_hub#edit_translation', as: :edit_translation
-  patch 'update-translation' => 'git_hub#update_translation', as: :update_translation
+
+  get 'translations/status' => 'translations_status#index'
+  get 'translations/edit' => 'git_hub#edit_translation', as: :edit_translation
+  patch 'translations/update' => 'git_hub#update_translation', as: :update_translation
```

Comment:
I wonder if it wouldn't be better to have `TranslationsController` with `#index`, `#edit` and `#udpate` actions.
The thought doesn't come from the fact that this would follow the convention (there's no `:id` anyway), but I feel like `TranslationsStatusController` will remain with a single action only.
Either way is ok though =)